tests/e2e-integration/e2e/sections/collection/Collection.spec.ts

```diff
@@ -122,7 +73,7 @@ describe('Collection Page', () => {
     })
   })
 
-  it('updates the query param when updateQueryParam is true', () => {
+  it('updates the page query param when navigating to another page', () => {
     cy.wrap(
       DatasetHelper.destroyAll().then(() => DatasetHelper.createMany(12)),
```

Comment:
I am getting an error from this test, I think because of a timing issue - the test runs before all 12 datasets are created, so it doesn't find page 2.

Reply:
Yep. I added some cy.wait() to wait for the datasets to be created.

Reply:
It works now, thanks!